admin-api.yaml

```diff
@@ -227,6 +227,160 @@ paths:
+                            type: integer
+                            description: Shows the amount of usage in minutes.
+                            example: 100
+                      video_mp4:
+                        type: object
+                        description: Returns the total value of mp4 video delivery usage in minutes for every day within the    interval you selected.
+                        properties:
+                          duration_minutes:
+                            type: integer
+                            description: Shows the amount of usage in minutes.
+                            example: 33
+                example:
+                  '2024-08-01':
+                    live_stream:
+                      duration_minutes: 51
+                    video_hls:
+                      duration_minutes: 100
+                    video_mp4:
+                      duration_minutes: 33
+                  '2024-08-02':
+                    live_stream:
+                      duration_minutes: 22
+                    video_hls:
+                      duration_minutes: 41
+                    video_mp4:
+                      duration_minutes: 124
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequest'
+              example:
+                constraint: required
+                detail: An attribute is invalid.
+                name: interval
+                title: Bad request.
+                total_problems: 1
+                type: 'https://docs.api.video/reference/invalid-attribute'
+      security:
+        - basicAuth: []
+  /projects/{project_id}/hosting-usage:
+    get:
+      tags:
+        - Usage
+      summary: Get hosting usage
+      description: This endpoint enables you to retrieve the cumulative amount of video minutes hosted in a specific project based on its `project_id`.
+      parameters:
+        - in: path
+          name: project_id
+          required: true
+          schema:
+            type: string
+          description: The ID of the project you want to retrieve usage data for.
+        - in: query
+          name: interval
+          required: true
+          schema:
+            type: string
+            enum:
+              - month
+              - week
+          description: Set the period of time that you want to retrieve hosting usage for. `month` returns data for the past 30 days, while `week` returns data for the past 7 days.
+      responses:
+        '200':
+          description: Successful response
+          content: 
+            application/json:
+              schema:
+                type: object
+                properties:
+                  '{date}':
+                    description: A date within the interval you selected in `YYYY-MM-DD` format.
+                    type: object
+                    pattern: '^\d{4}-\d{2}-\d{2}$'
```

Comment:
I added pattern because it does not break the OpenAPI validation, even though it is skipped. 🤷